### PR TITLE
Add SetResourceLimit function (fixes #107) 

### LINKED ISSUE #107: Put SetResourceLimit at package level
Would be nicer to stick SetResourceLimit at the package level since its global underneath anyways, then it would have to be called every time a MagickWand is created.

### PR DESCRIPTION
**Pull request by samt42**
Fix issue #107

### CODE REVIEW

**Comment by samt42** [comment]
Add ResourceType

File: imagick/magick_version.go [hunk 1]
```diff
@@ -68,3 +68,13 @@ func GetVersion() (version string, nversion uint) {
 	nversion = uint(cnver)
 	return
 }
+
+// Specify resource limit at package level.
+func SetResourceLimit(rtype ResourceType, limit uint64) bool {
+	ok := C.MagickSetResourceLimit(C.ResourceType(rtype), C.MagickSizeType(limit))
```
Comment (justinfx) [inline imagick/magick_version.go]: Make sure to run gofmt on your source files before committing.

File: imagick/magick_version.go [hunk 2]
```diff
@@ -68,3 +68,13 @@ func GetVersion() (version string, nversion uint) {
...
+    if C.int(ok) == 0 {
+        return false
+    } else {
+        return true
+    }
```
Comment (justinfx) [inline imagick/magick_version.go]: This could be simplified to

```
return C.int(ok) == 1
```

**Comment by justinfx** [comment]
Thanks for the contribution and addressing the notes!

**Pull request merged**